### ISSUE TITLE
Create a CircleCI rule that alerts the front-end review team if E2E tests don't include an axeCheck

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8781]
## Description
@brandonrapp 
Dan and I were talking with Jhonny about a CircleCI rule that could notify the front-end review team if E2E tests are created/edited without at least one [axeCheck for accessibility scans](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js). We are required to have these automated checks in our applications, so it would be good to alert team members if they're missing.

## Related Issues
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/8491

**Comment by 1Copenut** [comment]
This feature was rolled into the ESLint rule merged in this PR https://github.com/department-of-veterans-affairs/vets-website/pull/12667
